src/app/messages/en/attributionText.ts

```diff
@@ -62,15 +62,17 @@ export const defaultText = `
 </p>
 `;
 
-export const teaText = `
+export const specialText = `
 <p>
   Want to cite, share, or modify this book? This book is
   <a target="_blank" rel="noopener" href="https://creativecommons.org/licenses/by/4.0/">
    {bookLicenseName} {bookLicenseVersion}
   </a>
-  and you must attribute “Texas Education Agency (TEA)." The original material is available at: <a target="_blank"
-  rel="noopener" href="https://www.texasgateway.org/book/{teaBookName}">https://www.texasgateway.org/book/{teaBookName}
-  </a>. Changes were made to the original material, including updates to art, structure, and other content updates.
+  and you must attribute {copyrightHolder}.{displayOriginalMaterialInformation, select, yes {
+    The original material is available at: <a target="_blank" rel="noopener" href="{originalMaterialLink}">
+    {originalMaterialLink}
+  </a>.} no {}} Changes were made to the original material, including updates to art, structure, and
```

Comment:
maybe add another variable, or use that `displayOriginalMaterialInformation` one, for this text. i'd call it like `isDerivativeWork` or something